### PR TITLE
Use shared targeting from commercial-core

### PR DESCRIPTION
**Pull request by Jakeii**
## What does this change?
Use shared targeting and type from commercial core

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/build-page-targeting.ts [hunk 1]
```diff
@@ -62,58 +64,37 @@ const adManagerGroups = [
...
-	| 'crossword'
-	| 'audio'
-	| 'article';
-
-export type PageTargeting = PartialWithNulls<{
-	ab: string[];
-	af: 't'; // Ad Free
-	amtgrp: AdManagerGroup;
-	at: string; // Ad Test
-	bl: string[]; // BLog tags
```
Comment (domlander) [inline static/src/javascripts/projects/common/modules/commercial/build-page-targeting.ts:82]: Why are `bl` and others now removed?

Comment (domlander) [reply static/src/javascripts/projects/common/modules/commercial/build-page-targeting.ts:82]: Ahh do these live in `SharedTargeting`?

Comment (Jakeii) [reply static/src/javascripts/projects/common/modules/commercial/build-page-targeting.ts:82]: yeah exactly

Comment (arelra) [reply static/src/javascripts/projects/common/modules/commercial/build-page-targeting.ts:82]: +1 to adding all the other types from commercial-core in the same way - or we could export one PageTargeting type from core once everything has moved across.

**Review by domlander — APPROVED** [review]

**Review by chrislomaxjones — APPROVED** [review]
Nice!

**Pull request merged**